aten/src/ATen/native/native_functions.yaml

```diff
@@ -6303,6 +6303,11 @@
   dispatch:
     CPU, CUDA: _nested_compute_contiguous_strides_offsets
 
+- func: _nested_assert_expandable_to(Tensor tgt, SymInt[] src_shape, str msg) -> ()
```

Comment:
Can you make this a pure python dispatcher op please? :D

Reply:
Hmmm I need to access it from cpp though